tests/e2e/test_install_cluster.go

```diff
@@ -0,0 +1,51 @@
+// Copyright © 2023 Cisco Systems, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func testInstallZookeeperCluster() bool {
+	return When("Installing Zookeeper cluster", func() {
+		var kubectlOptions k8s.KubectlOptions
+		var err error
+
+		It("Acquiring K8s config and context", func() {
+			kubectlOptions, err = kubectlOptionsForCurrentContext()
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		kubectlOptions.Namespace = zookeeperOperatorHelmDescriptor.Namespace
+		requireCreatingZookeeperCluster(kubectlOptions, zookeeperReplicaCount)
+	})
+}
```

Comment:
**Note**: I will think about refactoring those because I'm not entirely sold on having these as separate test cases.

A test case should be something which run on its own does a full thing we want to test.

testInstall == installs the deployments
testClusterDeployment == -||- + deploys ZK and Kafka clusters
testProduceConsumeInternal == -||- + executes produce/consume with internal clients.

These can live as test steps == logical units for a bounded operation such as the requires.

This is something that becomes clear along the way, it's just some impression I started to have.

You can leave it this way for the PR but this needs immediate refactoring afterwards if we agree on something.